src/main.js

```diff
@@ -0,0 +1,66 @@
+#!usr/bin/env node
+let loader = require('taskcluster-lib-loader');
+let App = require('taskcluster-lib-app');
+let monitor = require('taskcluster-lib-monitor');
+let debug = require('debug')('events:main');
+let config = require('typed-env-config');
+let builder = require('./api');
+let taskcluster = require('taskcluster-client');
+let _  = require('lodash');
+
+// Create component loader
+let load = loader({
+  cfg: {
+    requires: ['profile'],
+    setup: ({profile}) => config({profile}),
+  },
+
+  monitor: {
```

Comment:
You'll also want to set the `enable` flag as in https://github.com/taskcluster/taskcluster-secrets/blob/master/src/main.js#L30

Reply:
how is `mock` different from `enable` ?

Reply:
`mock` keeps data in memory for testing. `enable` just turns it off entirely.